pkg/k8shandler/visualization.go

```diff
@@ -416,6 +512,13 @@ func newKibanaPodSpec(cluster *logging.ClusterLogging, kibanaName string, elasti
 		{Name: "kibana-proxy", ReadOnly: true, MountPath: "/secret"},
 	}
 
+	if proxyConfig != nil && len(proxyConfig.Spec.TrustedCA.Name) > 0 {
+		proxyCA := []v1.VolumeMount{
+			{Name: "proxytrustedca", MountPath: "/etc/kibana/proxy"},
```

Comment:
is the proxy configured to add this to its trusted CA bundle? 
re: https://bugzilla.redhat.com/show_bug.cgi?id=1766187

Reply:
@ewolinetz, thanks for mentioning the bz.  I realized I hadn't understood what needs to be done for the trusted CA bundle...  For now, the trusted CA bundle from the cluster proxy is just mounted like this. 
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/fluentd.go#L289-L291
Q1. Are we supposed to create a configmap in the fluentd and the kibana-proxy container as tested in https://github.com/openshift/cluster-logging-operator/issues/261???  Looking at the twin bz1766181 for alertmanager and its PR, it creates a configmap.  Is that the solution we should follow?
```
+      - configMap:
+          defaultMode: 420
+          items:
+          - key: ca-bundle.crt
+            path: tls-ca-bundle.pem
+          name: trusted-ca-bundle
+        name: trusted-ca-bundle
```
Q2. I'm confused at the term additionalTrustBundle.
`Install Openshift 4.2 with additionalTrustBundle`
Is it what implemented in the installer - [1]?  If so, it's "a PEM-encoded X.509 certificate bundle that will be added to the nodes' trusted certificate store."  Could there be a known name such as SecretName that we can use to mount in the kibana-proxy container?  And again, we are supposed to create a configmap with the bundle? 
[1] - https://github.com/openshift/installer/blob/master/docs/user/customization.md#additional-trust-bundle

Reply:
@ewolinetz, since this pr contains a test blocker fix, I'd like to merge it asap if possible.  And may I work on the trusted CA bundle fixes (for bz1766187 and global proxy trustedCA) in the separate PR?  Thanks!